src/tail/host.rs

```diff
@@ -1,18 +1,36 @@
+use hyper::server::conn::AddrIncoming;
+use hyper::server::Builder;
 use hyper::service::{make_service_fn, service_fn};
 use hyper::{Body, Method, Request, Response, Server, StatusCode};
+use tokio::sync::oneshot::Receiver;
 
-pub struct Host;
+pub struct Host {
```

Comment:
not sure what `Host` is supposed to be, worth adding a comment or make it a more descriptive struct name

Reply:
yeah it may be better to call this `LogServer`? it's the "localhost" you point your tunnel to that then receives the log requests from the trace worker. i almost think it is worth putting a really long comment description in `tail/mod.rs` and at the top of each component for this to outline the architecture.

Reply:
definitely worth the comment at least